src/calculateRentalCost.js

```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const rent = 40;
+  const discountMin = 20;
+  const discountMax = 50;
+
+  if (days < 3) {
+    return days * rent;
+  }
+
+  if (days < 7) {
+    return days * rent - discountMin;
+  }
+
+  return days * rent - discountMax;
```

Comment:
you repeat `days * rent` calculation 3 times, don't repeat yourself, store it in variable